Reducers/ScheduleReducer.js

```diff
@@ -0,0 +1,93 @@
+import {
+    FILTER_PERFORMANCES,
+    LOAD_PERFORMANCES_BEGIN,
+    LOAD_PERFORMANCES_SUCCESS,
+    LOAD_PERFORMANCES_END
+} from '../Actions/ScheduleActions/ScheduleActionTypes';
+
+let currentDate = new Date();
+let dateAfterWeek = new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate() + 7);
+
+const initialState = {
+    performances: [
+        { id: 1, title: "Курочка ряба", mainImage: "uri", startDate: '2019/02/9', beggining: '10:00' },
+        { id: 2, title: "Коза Дереза", mainImage: "uri", startDate: '2019/03/9', beggining: '10:00' },
+        { id: 3, title: "Колобок", mainImage: "uri", startDate: '2019/04/9', beggining: '10:00' },
+        { id: 4, title: "Семеро козенят", mainImage: "uri", startDate: '2019/02/5', beggining: '10:00' },
+        { id: 5, title: "Котигорошко", mainImage: "uri", startDate: '2019/02/3', beggining: '10:00' },
+        { id: 6, title: "Івасик-Телесик", mainImage: "uri", startDate: '2019/02/10', beggining: '10:00' },
+    ],
+    performancesBackup: [
```

Comment:
backend has possibility to filter performances by dates
what you think about usage of server side filtering?